modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -76,30 +76,50 @@ class IgniteSpec(metaclass=ABCMeta):
     This class is a basic Spec
     """
 
-    def __init__(self, service, jvm_opts, full_jvm_opts):
+    def __init__(self, service, jvm_opts=None, full_jvm_opts=None):
```

Comment:
May be sth like this?
```
def __init__(self, service, extra_jvm_opts=None, jvm_opts=None):
```
Also, may be good idea to raise exception if both of them is set, assertions may be ignored in certain circumstances.

Reply:
OK for exception.  

Not sure about the interpretation which is suggested for extra_jvm_opts vs. jvm_opts.

Do you mean the following:

*  _extra_jvm_opts_  are options to be **added** to default ones from the framework?
*  _jvm_opts_  are options to be applied "as is" ignoring the defaults from framework completely?
* Only one of two may be passed.

Reply:
```
def _init__(self, service, jvm_opts=None, merge_with_default=True):
....
```